docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -57,9 +57,7 @@ The main features of the Media Controller API include:
 
 - Setting abilities of the media controller server
```

Comment:
please change this title to managing abilities or sth similar and add proper sentences below about both server/client side managing of this feature

Reply:
Done.